### ISSUE TITLE
Feature Request: Ability to run against a remote server

### ISSUE THREAD

**Issue by C0deGyver** [EvotecIT/GPOZaurr#18]
I am not sure if this is even possible... But, it would be nice if it was possible to run this against a remote server. I have a server that I cannot update PowerShell on that I would like to check using its replacement that is a 2019 Std server.

**Comment by PrzemyslawKlys** [comment]
This works perfectly fine from any machine eith AD and GPO modules installed. I use it on Windows 10 or Windows 2016. Just domain joined with tools installed.

**Comment by C0deGyver** [comment]
I can understand that working as the system you ran it from is not an AD server so it has no GP's to look at. My point is to be able to run it from another server by doing something similar to Invoke-GPOZaurr -Server \\ServerHostname.

**Comment by PrzemyslawKlys** [comment]
I don't understand. GPOZaurr requires Active Directory to work. When you run it from Windows 10 it connects to closest Active Directory Server and requests the data required to get all GPOs for a given forest/domain. It doesn't even "connect" to AD server, it just queries AD and SYSVOL. It doesn't require Domain Admin to do "gathering" of the data. 

Can you explain your scenario where it doesn't work?

**Comment by C0deGyver** [comment]
I am trying to run it from another AD server; therefore it gives results for that server because it is the closest AD server.

**Comment by PrzemyslawKlys** [comment]
GPOZaurr scans forest/domain - it doesn't scan specific AD Server, as any GPO on server 1 should be on server 2, and server 3 and server 4 therefore GPOZaurr analyzes the whole forest/domain - and you shouldn't care for a specific server. If you need to find problems within specific servers use Testimo which targets forest/domain/DCS separately.

**Comment by C0deGyver** [comment]
I see. I was trying to find a broken GPO on the old server. It kept trying to apply it but could not. I have finally tracked down the registry key that pointed to the nonexistent GPO and deleted it. Now gpupdate no longer has any issues. I had been thinking that there might be a remnant of the GPO in SYSVOL somewhere that was causing the issue; but, after your post, I searched the registry for it and found it there.
